erpnext_documentation/www/docs/user/manual/en/accounts/deferred-revenue.md

```diff
@@ -1,43 +1,59 @@
 # Deferred Revenue
 
-**Deferred revenue refers to advance payments a Company receives for products or services that are to be delivered or performed in the future.** 
+**Deferred revenue refers to advance payments a Company receives for products or services that are to be delivered or performed in the future.**
 
 It is also known as unearned revenue.
 
 The company that receives the prepayment records the amount as Deferred Revenue on their balance sheet as a liability. Deferred revenue is a liability because it refers to revenue that has not been earned and represents products or services that are owed to a Customer. As the product or service is delivered over time, it is recognized as revenue on the income statement.
 
-## 1. How to use Deferred Revenue
+## 1. Configuring Deferred Accounting
+
+> Introduced in Version 13
+
+Before you start using deferred accounting you should be aware of the below settings which will give you more control over how you manage your deferred accounting
+
+<img class="screenshot" alt="Deferred Accounting Settings" src="{{docs_base_url}}/assets/img/accounts/deferred-accounting-settings.png">
+
+1. **Automatically Process Deferred Accounting Entry:** This setting is enabled by default. In case you don't want the deferred accounting entries to be posted automatically you can disable this setting. If this setting is disabled deferred accounting will have to be processed manually using [Process Deferred Accounting](/docs/user/manual/en/accounts/process-deferred-accounting)
```

Comment:
```suggestion
1. **Automatically Process Deferred Accounting Entry:** This setting is enabled by default. In case you don't want the deferred accounting entries to be posted automatically, you can disable this setting. If this setting is disabled, deferred accounting will have to be processed manually using [Process Deferred Accounting](/docs/user/manual/en/accounts/process-deferred-accounting)
```